test/chisel2.py

```diff
@@ -0,0 +1,229 @@
+"""
+A simple client that uses the Python ACME library to run a test issuance against
+a local Boulder server. Usage:
+
+$ virtualenv venv
+$ . venv/bin/activate
+$ pip install -r requirements.txt
+$ python chisel.py foo.com bar.com
+"""
+import json
+import logging
+import os
+import sys
+import signal
+import threading
+import time
+import urllib2
+
+from cryptography.hazmat.backends import default_backend
+from cryptography.hazmat.primitives.asymmetric import rsa
+
+import OpenSSL
+
+from acme import challenges
+from acme import client as acme_client
+from acme import errors as acme_errors
+from acme import jose
+from acme import messages
+from acme import standalone
+
+logging.basicConfig()
+logger = logging.getLogger()
+logger.setLevel(int(os.getenv('LOGLEVEL', 0)))
+
+DIRECTORY = os.getenv('DIRECTORY', 'http://localhost:14000/dir')
```

Comment:
Can you delete this line? It's set up for Pebble's WFE that runs on `14000` and uses `/dir` for its directory endpoint.